components/rhs_thread/rhs_thread.tsx

```diff
@@ -254,6 +274,14 @@ export default class RhsThread extends React.Component<Props, State> {
         this.setState({
             isScrolling: false,
         });
+
+        if (this.scrollbarsRef.current) {
```

Comment:
Capture scroll position upon stopping scrolling. This defaults to true as the thread is at the bottom when first rendered.

Reply:
Could we add a buffer of few pixels from the bottom for this? Like 10px or something so posts are auto scrolled if they are left a bit above the posts?

Reply:
Based on limited testing with a few input devices and browsers, a "slight" scroll can easily be 150px. This isn't an exact science but I felt 200px seemed reasonable to assume a user "meant" to be at the bottom.

Reply:
0/5 but 200px seems a lot to me, I think we do about 10px in center channel.